build.mjs

```diff
@@ -96,10 +96,19 @@ const esbuildSassPlugin = {
   },
 };
 
-const renderTemplate = async (template) => {
+const renderTemplate = async (template, language) => {
+  const localePath = path.join(__dirname, "src", "locales", language + '.json');
+  const t = JSON.parse(readFileSync(localePath));
```

Comment:
Since this is an async function, consider awaiting the promise version of readFile.